app/models/calagator/source/parser/meetup.rb

```diff
@@ -7,15 +7,19 @@ class Source::Parser::Meetup < Source::Parser
   def to_events
     return fallback unless Calagator.meetup_api_key.present?
     return unless data = get_data
+    start_time = Time.at(data['time']/1000).utc
+    group_topics = data['group']['topics']
+    topics_string = group_topics.map{ |t| t['name'].downcase }.join(', ').insert(0, ', ') unless group_topics.empty?
```

Comment:
This method is getting pretty complex, especially this line. I'd consider extracting some private methods, perhaps.